### PR TITLE
Update npm guide to include core-util-is error

### PR DESCRIPTION
**Pull request by a-ashbeck**
Let me know if you think this needs more detail. I tried to make non-SiS specific. The steps listed here are the ones that appeared to work, not those that lead up to the solution.

### CODE REVIEW

File: tools/npm/README.md [hunk 1]
```diff
@@ -23,3 +23,50 @@ cd my_project
...
+    at require (module.js:384:17)
+    at Object.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/readable-stream/lib/_stream_readable.js:41:12)
+    at Module._compile (module.js:460:26)
+    at Object.Module._extensions..js (module.js:478:10)
+    at Module.load (module.js:355:32)
+    at Function.Module._load (module.js:310:12)
+    at Module.require (module.js:365:17)
+```
+
+If you recieve this error, first attempt to delete your local copy of the repo in question and re-clone it. If that does not work, complete the following steps:
```
Comment (cbrenner04) [inline tools/npm/README.md]: @a-ashbeck this line is too long

**Comment by ericcf** [comment]
@a-ashbeck the "build" for this repository runs a MarkDown linter to ensure consistent formatting. It failed because a couple of your lines of text are over 80 characters. Do you mind fixing that and updating this PR? Thanks!

File: tools/npm/README.md [hunk 2]
```diff
@@ -23,3 +23,50 @@ cd my_project
...
+brew install node012
+```
+
+```
+sudo chown -R <user> /usr/local/lib/node_modules
+```
+
+If you are still receiving an error, redo the first two steps.
+
+If running a Cordova hybrid app, make sure to create a www directory at the top level of the repo:
```
Comment (cbrenner04) [inline tools/npm/README.md]: @a-ashbeck this line is too long

**Comment by a-ashbeck** [comment]
@ericcf @cbrenner04 Shortened the line lengths.

**Pull request merged**